runtime/onert/core/include/compiler/LoweredGraph.h

```diff
@@ -36,7 +36,8 @@ namespace compiler
 class LoweredGraph
 {
 public:
-  LoweredGraph(const ir::Graph &graph, const compiler::CompilerOptions &options);
+  LoweredGraph(const ir::Graph &graph, const compiler::CompilerOptions &options,
+               std::shared_ptr<util::TracingCtx> tracing_ctx);
```

Comment:
What about putting `tracing_ctx` in `CompilerOptions`?

Reply:
Do you mean the following?

1. creating `TracingCtx` at `nnfw_create_session()` (as in this PR)
2. pass `TracingCtx` to `compilerOption` and let `compilerOption` has ownership of `TracingCtx`

Reply:
While rewriting this draft, I decided to put `TracingCtx` into `nnfw_session` (not in `CompilerOptions`)
But with your different comment(https://github.com/Samsung/ONE/pull/4903#discussion_r531352334)
I think what you meant is
1) Let `nnfw_session` has ownership of `TracingCtx`
2) Store ref of `TracingCtx` also into `CompilerOptions` to reduce the number of params.

I will fix like the above.

Reply:
fixed as described in the above comment.